voxblox_ros/src/esdf_server.cc

```diff
@@ -120,7 +117,11 @@ void EsdfServer::updateMesh() {
     const bool clear_updated_flag_esdf = false;
     esdf_integrator_->updateFromTsdfLayer(clear_updated_flag_esdf);
   }
-  publishAllUpdatedEsdfVoxels();
+  if (publish_pointclouds_) {
+    publishAllUpdatedEsdfVoxels();
+  } else if (publish_traversable_) {
```

Comment:
why is this an either or? The flags don't really tell and not sure this is intuitive.

Reply:
Good catch!